hoomd/state.py

```diff
@@ -34,15 +40,29 @@ def _create_domain_decomposition(device, box):
 
 
 class State:
-    R""" Simulation state.
-
-    Args:
-        snapshot
+    R"""The state of a `hoomd.Simulation` object.
```

Comment:
Should we use `"""` instead of `R"""` everywhere? I recall running into problems with the `R"""` in some tools, though now I can't remember which. PEP257 recommends `r"""` when using backslashes.

Reply:
I tried this with pydocstyle without any problems, but I can remove it or use `r"""` on your preference.

Reply:
I think lowercase `r"""` is preferable. As for raw vs. normal strings, the case I always run into that requires raw strings is when documenting the parameters `*args` and `**kwargs`. There is some magic in how you place the backslashes so that Sphinx doesn’t try to render the asterisks as italic/bold font. I have commented about this before and can find examples/demos if needed. Otherwise I prefer non-raw strings wherever possible.